Code/GraphMol/RWMol.h

```diff
@@ -49,6 +49,9 @@ class RDKIT_GRAPHMOL_EXPORT RWMol : public ROMol {
   RWMol(const RWMol &other) : ROMol(other){};
   RWMol &operator=(const RWMol &);
 
+  //! construct a molecule from a pickle string
```

Comment:
This definition isn't necessary. The ctor is inherited from ROMol.
There are even tests that ensure it works. Here's one:
https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/testPickler.cpp#L840